packages/react-select/src/components/Select/Select.types.ts

```diff
@@ -0,0 +1,41 @@
+import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
+
+export type SelectSlots = {
+  /*
+   * Wrapper for both the select and icon, renders as a `<span>`.
+   * The `className` and `style` props on `<Select>` are applied to this slot;
+   * All other top-level props are applied to the primary slot, `select`.
+   */
+  root: NonNullable<Slot<'span'>>;
+
+  /** Primary slot: the actual `<select>` element */
+  select: NonNullable<Slot<'select'>>;
+
+  /** the icon, typically a down arrow */
+  icon: Slot<'span'>;
+};
+
+export interface SelectCommons {
+  /**
+   * Matches the Input sizes
+   * @default 'medium'
+   */
+  size?: 'small' | 'medium' | 'large';
+
+  /**
+   * If true, the Select will have an inline `display`, allowing it to be inline with other content.
+   * By default, Select has block layout.
+   * @default false
+   */
+  inline?: boolean;
+
+  /**
+   * Controls the colors and borders of the Select.
+   * @default 'outline'
+   */
+  appearance?: 'outline' | 'underline' | 'filledDarker' | 'filledLighter';
```

Comment:
how does darker and lighter work with high contrast or dark themes? Is there a need for color specific appearances?

Reply:
They don't affect high contrast, but there are designs in the Figma spec for them in dark theme :). The appearance prop and what it controls is pretty much the exact same as for Input.

Reply:
just worried about a future where the design changes a bit and 'darker' is no longer 'darker'. Curious if there was a discussion on how we named those variants in our system? Obviously it seems that it's used a bunch across different components, so would want to ensure consistency across all of them.